Formula/ninja.rb

```diff
@@ -21,6 +21,11 @@ class Ninja < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "python@3.8"
+
+  on_linux do
+    depends_on "re2c" => :build
```

Comment:
Is `re2c` even a build dependency? It was [added](https://github.com/Homebrew/homebrew-core/pull/60467) and [removed](https://github.com/Homebrew/homebrew-core/pull/60512) recently and I believe it's only needed if modifications are made to a couple particular source files during development.

Reply:
I added it because it complained during the build that it was needed. This is why I added it only on Linux, as it refused to build without it. I did not know why it was needed only on Linux and not on Mac.

Maybe because I am modifying the "CMakeLists.txt" file?

Reply:
Do you have the error log from when you tried to build it? I wouldn't think changing CMakeLIsts.txt in the way you are would cause re2c to be required.

Reply:
Weird, I could not reproduce the error log. But I am sure it asked for re2c at one point. As I can't reproduce, I will drop that dependency.